docs/design/unified-design.md

```diff
@@ -1,17 +1,29 @@
 ## Background
 
-As machine learning workloads become increasingly complex, there is a growing need to support hybrid training paradigms that combine elastic training with other processing strategies. Traditional elastic training frameworks are optimized for scaling model training dynamically, but often lack the flexibility to integrate with heterogeneous tasks such as data preprocessing, reinforcement learning environments, or custom compute-intensive operations.
+As machine learning workloads become increasingly complex, there is a growing need to support hybrid training paradigms
```

Comment:
need a specified doc(unified_elastic_training_guide.md) for elastic training using under: docs/tutorial/unified/